recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "tiledbsoma" %}
-{% set version = "1.2.7" %}
-{% set version_r = "0.0.0.9024" %}
-{% set sha256 = "58942b6a6c6a6a6b153c58c4879596d32dbe7b0551f44c635e297633fba62f0b" %}
+{% set version = "1.3.0" %}
+{% set version_r = "0.99.0" %}
```

Comment:
The version of the R package has been bumped from 0.99.0 to 0.99.1

https://github.com/single-cell-data/TileDB-SOMA/blob/1.3.0/apis/r/DESCRIPTION#L7